### PR TITLE
Updated tinysearch

### PR DESCRIPTION
**Pull request by antoine62**
I fixed a small bug that create an unexpected exeption when we do .tinysearch .. (or .tinysearch something/something. Well, it work, i was sure it wasnt) 
(There could be a better way to do that.) Tanks deadphoenix8091

And, user can now search apps with a space in their name, like homebrew launcher. (Right now, user can type homebrew_launcher to get it.)

### CODE REVIEW

**Comment by deadphoenix8091** [comment]
It's probably better if I implement this in the search api.

**Comment by antoine62** [comment]
It would still require some part of my code to work with space, and the bug fix is still require

**Comment by antoine62** [comment]
DO NOT MERGE IT NOW! Things are broken

**Comment by deadphoenix8091** [comment]
Looks good to me now 👍

File: cogs/assistance.py [hunk 1]
```diff
@@ -857,13 +858,16 @@ def check_console(self, message, channel, consoles):
 
     @commands.command(aliases=["tinydbsearch"])
     @commands.cooldown(rate=1, per=15.0, type=commands.BucketType.channel)
-    async def tinysearch(self, ctx, app: str = ""):
+    async def tinysearch(self, ctx, *app):
         """Search for your favorite homebrew app in tinydb"""
```
[the remark this answers is not in the document]
Comment (antoine62) [reply cogs/assistance.py]: I dont know, i will try that tomorrow (its late here), If you want, you should be able to edit my port branch.

**Comment by antoine62** [comment]
The problem is:
.tinysearch a/../.. create an error.
What should i do? 
@FrozenChen

**Comment by antoine62** [comment]
I also added the lastest change, i dont know if its the good way to do that.

**Comment by FrozenChen** [comment]
Looks good to me

**Pull request merged**